vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/BasicUseIT.java

```diff
@@ -131,7 +132,24 @@ public void crudAndGeneratedGridReactToThemeVariantChanges() {
         Assert.assertNotEquals("no-border", grid.getAttribute("theme"));
     }
 
+    @Test
+    public void newEventItem() {
+        CrudElement crud = $(CrudElement.class).waitForFirst();
+        $("#newEventListener").onPage().first().click();
+        // we need to do this trick because the way `New` events are logged
+        crud.getNewItemButton().get().click();
+        crud.getEditorCancelButton().click();
+        Assert.assertEquals(
+                "Cancel: Person{id=0, firstName='firstName', lastName='lastName'}",
+                getLastEvent());
+    }
+
     private ButtonElement getTestButton(String id) {
         return $(ButtonElement.class).onPage().id(id);
     }
+
```

Comment:
This is overriding a method from AbstractParallelTest with the same implementation. It could be removed

Reply:
There was no such method in `AbstractParallelTest`.

Reply:
This comment is still valid, the method that is being overridden is here: https://github.com/vaadin/flow-components/blob/master/vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/AbstractParallelTest.java#L7-L9